WcaOnRails/app/models/person.rb

```diff
@@ -11,6 +11,13 @@ class Person < ActiveRecord::Base
 
   scope :current, -> { where(subId: 1) }
 
+  scope :in_region, lambda { |region|
```

Comment:
**lambda**: My problem is with using `lambda { |parm|` instead of `-> (parm)`. I don't care much about `{}` vs `do end`. **However**, to return values, a lambda has to use `{}`, because of some operator precedence thing, so many people use braces for all lambdas.

**continent ids**: I thought about looking at underscores, but aside from making the code more fragile in theory (in practice, the set of continents is quite stable), I don't think that could make this code faster or shorter.
   But after sleeping on it, I don't like using the internal cache structure  (`c_all_by_id`) outside the class, so I made a `Continent.country_ids(continent_id)` function.